torch/_inductor/codecache.py

```diff
@@ -102,6 +102,9 @@ def cpp_compiler_search(search):
     for cxx in search:
         try:
             if cxx is None:
+                # Do not attempt to install gcc-12 by default
+                if os.getenv("TORCH_INDUCTOR_INSTALL_GCC_VIA_CONDA"):
+                    continue
```

Comment:
May I know the default value? If it is FALSE by default and the user does not install a compiler, it will raise `InvalidCxxCompiler`. Is it as expected?

Reply:
Good point, should be inverse. But default value should be False, as Soumith suggested we should not be messing with users install without explicit permission